specification/advisor/resource-manager/Microsoft.Advisor/stable/2022-10-01/advisor.json

```diff
@@ -0,0 +1,1766 @@
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "description": "The number of suppressions per page if a paged version of this API is being used.",
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          },
+          {
+            "name": "$skipToken",
+            "in": "query",
+            "description": "The page-continuation token to use with a paged version of this API.",
+            "required": false,
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Successfully got all suppressions in a subscription.",
+            "schema": {
+              "$ref": "#/definitions/SuppressionContractListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ArmErrorResponse"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "ListSuppressions": {
+            "$ref": "./examples/ListSuppressions.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/predict": {
+      "post": {
+        "tags": [
+          "Prediction"
+        ],
+        "summary": "Predicts a recommendation.",
+        "operationId": "Predict",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "name": "predictionRequest",
+            "in": "body",
+            "description": "Parameters for predict recommendation.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/PredictionRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK. Successfully obtained prediction.",
+            "schema": {
+              "$ref": "#/definitions/PredictionResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ArmErrorResponse"
+            }
+          }
+        },
+        "deprecated": false,
+        "x-ms-examples": {
+          "Predict": {
+            "$ref": "./examples/Predict.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Advisor/advisorScore": {
+      "get": {
+        "tags": [
+          "AdvisorScore"
+        ],
+        "description": "Gets the list of advisor scores.",
+        "operationId": "AdvisorScores_List",
```

Comment:
Likely be x-ms-pageable

Reply:
@weidongxu-microsoft : The advisor score returns is a list of 5 score categories : Cost, Performance,Security,Operational excellance, Reliability and is static- hence not tagging it with x-ms-pageable.

Reply:
Then why does response schema has `nextLink`?

Reply:
Good point. Removed from the  schema